From 2b98d5d91d95087d8a96d6450fa96414ed05ba5c Mon Sep 17 00:00:00 2001
From: Andrea Bolognani <abologna@redhat.com>
Date: Mon, 23 May 2022 10:31:02 +0200
Subject: [PATCH] apparmor: Allow locking AAVMF firmware

We already allow this for OVMF.

Closes: https://gitlab.com/libvirt/libvirt/-/issues/312
Signed-off-by: Andrea Bolognani <abologna@redhat.com>
Reviewed-by: Martin Kletzander <mkletzan@redhat.com>

X-Backport-Note: 07af71ad has not landed yet and since it sorted all entries
                 there is some non-functional noise
Origin: backport, https://gitlab.com/libvirt/libvirt/-/commit/2b98d5d91
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1989078
Last-Update: 2023-01-09

---
 src/security/apparmor/libvirt-qemu | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/src/security/apparmor/libvirt-qemu
+++ b/src/security/apparmor/libvirt-qemu
@@ -91,7 +91,7 @@
   /usr/share/misc/sgabios.bin r,
   /usr/share/ovmf/** rk,
   /usr/share/OVMF/** rk,
-  /usr/share/AAVMF/** r,
+  /usr/share/AAVMF/** rk,
   /usr/share/qemu-efi/** r,
   /usr/share/slof/** r,
 
